superset-frontend/plugins/plugin-chart-echarts/src/Heatmap/controlPanel.tsx

```diff
@@ -224,6 +225,31 @@ const config: ControlPanelConfig = {
         ['y_axis_format'],
         ['x_axis_time_format'],
         ['currency_format'],
+        [<ControlSubSectionHeader>Border</ControlSubSectionHeader>],
+        [
+          {
+            name: 'element_border',
+            config: {
+              type: 'ColorPickerControl',
+              label: t('Border color'),
+              renderTrigger: true,
+              description: t('Select border color for Heatmap elements'),
+              default: { r: 255, g: 255, b: 255, a: 1 },
+            },
```

Comment:
Could you also move the configuration after `linear_color_scheme` as it makes sense to be close to the color scheme?

```suggestion
          {
            name: 'border_color',
            config: {
              type: 'ColorPickerControl',
              label: t('Border color'),
              renderTrigger: true,
              description: t('The color of the elements border'),
              default: { r: 0, g: 0, b: 0, a: 1 },
            },
```